.github/ISSUE_TEMPLATE/decoration_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Decoration Request
+description: Suggest a new decoration to be added to the package
+title: "[Decoration]: "
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Write the decoration you want to add
+      placeholder: >
+        ex: ▌│█║▌║▌║ 
+    validations:
+      required: false
```

Comment:
I think this item should be `true`

Reply:
fixed in [7dcf2a6](https://github.com/sepandhaghighi/art/pull/265/commits/7dcf2a62162ca0c223226e970ba03f5bd69c7c85).